src/cli.js

```diff
@@ -456,10 +456,13 @@ function cli(inputArgs) {
                 template: false
             };
 
-            if (args['link-to'])
-                wwwCfg.link = true;
-            else if (args.template)
+            
+            if (args.template) {
+                wwwCfg.template = true;
+            } else if (args['copy-from']) {
+                logger.warn('Warning: --copy-from option will be deprecated on December 25, 2016');
```

Comment:
It would be nice to suggest using `--template` instead in the deprecation message